internal/provider/resource_rediscloud_subscription.go

```diff
@@ -591,13 +591,21 @@ func buildSubscriptionCreatePlanDatabases(planMap map[string]interface{}) []*sub
 	replication := planMap["replication"].(bool)
 	// Add modules to the request
 	var modules []*subscriptions.CreateModules
+	// If there is an incompatible module, then allocate it to a separate db.
+	var incompatibleModule string
```

Comment:
I think this may be more extensible for future changes if "incompatibleModule" was made into a string list "incompatibleModules" and then these could be looped through at line 623. This would mean if other incompatible modules are added in future then they can simply be added to this list